scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -408,10 +408,19 @@ if [[ -z "$projectName" ]]; then
       case $runLatest in
       [nN]*)
         allKnownVersions=$(get_all_known_versions)
+        optionCount=$(wc -l <<< "$allKnownVersions")
         echo
-        echo "Which version to you want to run? (ctrl + c to quit)"
-        select preferredRelease in $allKnownVersions; do
-          break
+        echo "Enter 1 through ${optionCount} or 'ctrl + c' to quit: "
```

Comment:
```suggestion
        echo "Select a version by entering 1 through ${optionCount} or 'ctrl + c' to quit: "
```
I'm skeptical of this message as it doesn't say what it does. This might be trivial to an existing user but not so straightforward to a newcomer.

Reply:
agreed - thanks!